src/Microsoft.ML.PCA/PCACatalog.cs

```diff
@@ -65,6 +71,12 @@ public static PrincipalComponentAnalysisEstimator ProjectToPrincipalComponents(t
         /// </summary>
         /// <param name="catalog">The anomaly detection catalog trainer object.</param>
         /// <param name="options">Advanced options to the algorithm.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[RPCA](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSampleWithOptions.cs)]
```

Comment:
>RandomizedPcaSampleWithOptions [](start = 119, length = 30)

i don't think this exists

Reply:
Added. Thanks!

---
In reply to: [261047854](https://github.com/dotnet/machinelearning/pull/2780#discussion_r261047854) [](ancestors = 261047854)